public/Projects/VaibhavPalindromeCheck/index.html

```diff
@@ -6,72 +6,73 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Palindrome-Checker</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css"
```

Comment:
I guess keeping this bootstrap style was a good idea. 
The previous navbar and button style was looking better.
However if you can make them look better without bootstrap using CSS only then that will be nice as well. Keeping this upto you!